### DISCUSSION TITLE
I added gamepad support and other Improvements

### DISCUSSION THREAD

**Discussion by fgilde** [Babyhamsta/Aimmy#1440] [Show and tell]
Hi all,

I found this repo 2 days ago, and because I created a similar project a long time ago, I think it's very interesting. I have made several changes and improvements, including:

Gamepad Support:

- Added Gamepad read support
- Working on Gamepad sending support

General Improvements:

 - Added a global active switch with an indicator and toggle key
 - Added settings for the gamepad before it counts as pressed
 - Created an FOV Opacity slider

Auto Trigger Enhancements:

 - Made the auto trigger completely independent of the Aim Assist with separate center and head area detection in the AIManager
 - Added an additional Send command for the trigger, allowing automatic pinging of enemies
 - Custom head area to setup when the trigger should hit. You can also configure leg shots with this feature
 - Option to specify the minimum time the trigger key must be held down before the trigger is executed.
 
 ESP Improvements:

 - Added option to show relative trigger head area

I have only worked on this for 3 days now, so nothing is perfect right now. But In the future, I plan to:

Completely change the trigger system to allow multiple triggers. This will enable you to add an Auto Trigger, set up the command to send, define the area, key binding, etc., allowing you to create a comprehensive workflow.
Add Auto rapid fire.

However, my current version looks like this:

![image](https://github.com/Babyhamsta/Aimmy/assets/11070717/e9fbabce-4be2-49e2-a100-163d62b4e26a)

![image](https://github.com/Babyhamsta/Aimmy/assets/11070717/4ff4f0b1-ce6a-493c-8d76-a1bee6c5eac8)

![image](https://github.com/Babyhamsta/Aimmy/assets/11070717/2e0d6838-9f3c-4266-918e-25c6f45b1f7e)

![image](https://github.com/Babyhamsta/Aimmy/assets/11070717/d466023e-a489-40a4-975f-54eec335314b)

Ah... and when it is Active it looks like this, so its easy to see if it is currently running

![image](https://github.com/Babyhamsta/Aimmy/assets/11070717/3b2a952f-dbce-4ce2-919b-dbdca9808951)

If you want to test it I have here the current state. If you want to use it ensure you copy the DirectML.dll and the onnxruntime.dll into the directory where you unzip this 

[TotallyNotAimmyV2_Ext.zip](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.1/1.0.0.1.zip)

My general question is to the Main team and @Babyhamsta would you like to have this changes as well? Then I would clean up a bit and try to create a fork PR

BTW and offtopic: I gave you Star on github, maybe you want to do this for my main Project as well ;-) https://github.com/fgilde/MudBlazor.Extensions/

**Comment by Harshjeett** [comment]
Can you make the auto trigger not be just left click, it would be helpful if you can choose which key it triggers instead of just left click.

**Reply by fgilde** [reply]
You currently can with the option "Trigger Additional Send" but ganepad sending is currently not supported. Currently work on that

**Reply by Harshjeett** [reply]
Thanks for the update! really like the toggle on and off ability. I have noticed trigger delay isn't working as it's supposed to be, currently its triggering at 0 ms even with delay set to 1 second.

**Reply by fgilde** [reply]
[TotallyNotAimmyV2.zip](https://github.com/user-attachments/files/16160050/TotallyNotAimmyV2.zip)

Fixed that here for you. Also this version has new feature for specify the minimum time the trigger key must be held down before the trigger is executed.

**Reply by edwardgon115-crypto** [reply]
can i use aimmyV2 on controller?

**Reply by sylvester-thsly** [reply]
very nice project :)

**Comment by ghost** [comment]
not bad modify
+rep

**Comment by NeedlessPage819** [comment]


**Reply by fgilde** [reply]
As I described if you want to use it ensure you copy the DirectML.dll and the onnxruntime.dll into the directory where you unzip this. I wasn't able to add this files because of too large zip for GitHub

**Comment by velezvfx** [comment]
im trying it on COD and for me there is no movement input sent to the app . the auto trigger works fine and everything else but the aim doent move at all . ive selected the applicateion but still no input sent . also im runing the application in admin mode

**Comment by fgilde** [comment]
Hi, I continued working on Aimmy. If you want to use my version you need to know that the config is no longer compatible that means you cant use my exe in the old directory anymore.  With my changes also a new Launcher that also works as Installer and updater is available. If I make a new build in my repo you will automatically informed about a new version that you can easially update with one click

![image](https://github.com/user-attachments/assets/38316ac0-9a91-4535-a3dd-20e3a7f00c2f)


## Some but not all of the New Features
- **Screen Selection for Capturing**: Users can now select a specific screen to capture, providing more control over which display content is captured in multi-screen setups.
- **Process Selection for Capturing**: Capture content from a specific process, allowing for more targeted and efficient capturing.
- **Additional Trigger Command**: Support for additional trigger commands via mouse, keyboard, or gamepad.
- **Additional Trigger Intersection Options**: Configure precise conditions for when additional triggers should be activated, enhancing control over trigger mechanisms.
- **Additional Trigger Key Hold**: An additional trigger key must be held to activate the trigger, adding a layer of control to prevent accidental triggering.
- **Desktop Graphic Context for ESP**: Option to use the desktop graphic context for ESP to improve performance.
- **Min Time Trigger Key**: Set the minimum time the trigger key must be held before the trigger activates, allowing for more precise timing control.
- **Configurable Toggles with Optional Keybinding**: All toggles are now configurable with optional keybindings, making it easier to switch each toggle on or off.
- **Model and Config Keybinding**: Quickly and easily switch models or configurations using keybindings, enhancing workflow efficiency.
- **Config Stores Last Model**: Configurations now save the last model used, ensuring consistency when a specific model is needed.
- **Theming with AI Tracking**: Select a specific theme when AI Tracking is active, allowing for a more customized user experience.
- **Gamepad Send with XInput Emulation, vJoy, or ViGem**: Enhanced support for gamepads with options to send inputs via XInput emulation, vJoy, or ViGem.
- **Improved Auto Update with Release Notes Display**: The auto-update feature now includes a display of release notes, keeping users informed of new features and changes.
- **Support for Gamepad DPad, Start, and Select**: Full support for gamepad DPad, Start, and Select buttons, improving compatibility with a wider range of gamepads.
- **Enhanced Key Binding Display**: Improved display for key bindings, making it easier to see and configure control settings.
- **New Installer and Launcher**: The installer and launcher have been revamped in a new, improved application for a smoother installation and launch experience.
- **Rapid Fire Feature**: A new rapid fire feature has been added, providing fast, repeated input activation for supported devices.
- **Magnification tool**: New tools section with a configurable magnification tool.


[Download Zip](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.1/1.0.0.1.zip)
[Download Installer](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.1/Installer.exe)

**Reply by AL99117** [reply]
After opening the program, the window is invisible and transparent. How should I solve it?

**Reply by fgilde** [reply]
wich operating system do you use? Not sure what the problem is maybe the hiding for captures from other programs to avoid bans.

Please ensure a clean empty directory where you extract the zip. Or use the installer and ensure you use an new empty directory. Also then use the Launcher.exe to start the application

**Reply by AL99117** [reply]
Win11，1.0.1 The version can be displayed normally, but the program window cannot be seen after 1.0.3 is launched, and the directory is clean. Launcher.exe is also used to launch the application. It should be a UI bug.

**Reply by fgilde** [reply]
But you can see the process is running? In taskbar or whatever? maybe its on another screen or something

**Reply by fgilde** [reply]
If wrong restore the case for the positioning you can delete each file in you AppData config folder "C:\Users\<User>\AppData\Roaming\AI-M"

**Reply by AL99117** [reply]
It can be used normally

**Comment by AL99117** [comment]
Can you add Chinese languages

**Reply by fgilde** [reply]
Sure

**Reply by AL99117** [reply]
thank you very much

**Comment by fgilde** [comment]
Done! Now more or less this are the newest Features in my 1.0.0.4


## General Improvements

- **UI Elements Hidden During Capture:**  
  All user interface elements are now hidden during screen captures. This allows seamless streaming with software like OBS, while keeping features like Field of View (FOV) and ESP hidden from the viewer.

- **Multi-Language Support:**  
  We've introduced support for multiple languages. You can now select your preferred language from the settings menu, including:  
  - English  
  - German  
  - French  
  - Italian  
  - Spanish  
  - Turkish  
  - Russian  
  - Ukrainian
  - Chinese

## Magnifier Enhancements

- **Improved Magnifier to work with Auto Trigger or AIM:**  
  The magnifier now works seamlessly with the auto-trigger feature, ensuring it no longer interferes with the capture process.

- **Automatic Centering:**  
  The magnifier will now automatically position itself in the center of the captured area or process for better usability.

## Capture Functionality

- **New Capture Border Setting:**  
  A new option is available in the settings menu to show a border around the current captured area or process.

- **Bug Fixes and Stabilization:**  
  - Fixed known issues (Part 1).
  - ESP and FOV now function correctly across all screens or processes, including those on different monitors.
  - **Stability Improvements:** If a process is terminated while capturing, or if the tool starts before the process is ready, it will now wait and automatically reattach after the process is launched or relaunched.

**Comment by venomissionary** [comment]
How do i bind the controller trigger to AimKeyBind? or is it not working yet?

**Reply by fgilde** [reply]
Sure the keybind will work, just ensure gamepad is connected and record a binding then press the desired key on controller. But the aiming itself currently not working with controller. Aiming still just moves the mouse

**Comment by BugScouter** [comment]
Have you considered integrating CUDA into your build? I've observed significant performance improvements when utilizing CUDA.

**Reply by AL99117** [reply]
How much improvement has been made in reasoning speed? It feels pretty fast now.

**Reply by venomissionary** [reply]
It's hardly a difference to me. I dont think it matters if  you got a fast cpu in the first place.

**Reply by fgilde** [reply]
currently I just seperate each as a full async service that nothing blocks something. for example if the aim is slow because of bad logic it will not make any problem on the trigger because this is a seperate async action. And this depends on your cpu. if you have power you will have a fast async thread.

**Comment by venomissionary** [comment]
is there an update to the program?

**Reply by fgilde** [reply]
yes now the CUDA version is available and working

**Comment by AL99117** [comment]
Can you optimize the problem of twitching back and forth when there are two enemies in the range?

**Reply by fgilde** [reply]
You can specify the Detection Area type and set this to mouse and/or additionally you can set a smaller dynamic FOV and use your AIM Key as Dynamic FOV Keybind

**Comment by fgilde** [comment]
Hi @all this is maybe my last release because of missing time in the Future. But also maybe not who knows. 
However I created a new version With the following Features. 
Please if you like it leave a comment or a like or something that I can see that somebody enjoy my changes

## CUDA Support
 **NOTICE:** *Important: The updater from 1.0.0.4 or older will not install the Cuda version, but further updates will install the same version as initial. So to switch from directML version to cuda you need to manually download the cuda release or use the newest installer*


 Cuda Support is in general available now. 
 Please notice that the CUDA is build in, in a different installation. That means if you want to use CUDA ensure you install/copy the CUDA version from the release assets. In the Installer you can select the Cuda version, if you download the zip ensure to downlod the Release_<version>_cuda.zip
If you want to use the CUDA version verify that CUDA and cuDNN are correctly installed
Ensure that:

 - CUDA Toolkit: The correct version of CUDA Toolkit is installed. You can verify this by running the command nvcc --version in your terminal to check which version of CUDA you have installed.
 - cuDNN: cuDNN must be installed and should be compatible with your CUDA version. You can download it from the [NVIDIA cuDNN site](https://developer.nvidia.com/cudnn) and follow the installation instructions.
Make sure that both the CUDA Toolkit and cuDNN paths are added to your system's PATH environment variable. 

After launching the Application you can see in the bottom right if cuda is working, otherwise CPU fallback is used

 - ![image](https://github.com/user-attachments/assets/3d857e61-695c-427f-a06f-56e918ba26f7)


## Multi Trigger Support
 - The Auto Trigger is now a editable collection where you can add as many trigger as you want.
 - Each trigger can have his own Actions/Commands to send and an individual area and condition for the execution
 - Trigger can also send a sequential sequence of keys to execute. For each Key you can configure a delay
 - Trigger can record a whole sequence to send
 - Trigger can work without a detection for example to build a rapid fire behaviour
 - A trigger can now have multiple Keys that needs to be pressed before trigger will execute.
    - For each key you can specify a time. That means the key needs to be hold for the time value before trigger will executed
    - You can specify if one of the keys or all needs to be pressed before trigger will execute
 - A trigger can now have multiple Anti Trigger Keys. If configured the Trigger will not execute if any/all of the keys here is/are pressed.
 - A trigger has now the option for a breaktime (Cooldown) If you build a auto ping with this you can ensure that it will not ping the whole time for example

![image](https://github.com/user-attachments/assets/fa31b7a8-22c3-463e-aacd-0d4c3c11d7aa)

 
## Aim Assist
 - For the AimAssist you can now specify as many aim keys you want

- **Improved Magnifier to work with Auto Trigger or AIM:**  
  The magnifier now works seamlessly with the auto-trigger feature, ensuring it no longer interferes with the capture process.

# Download

[CUDA VERSION](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.5/1.0.0.5_cuda.zip)
[Default DirectML Version](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.5/1.0.0.5.zip)
[Installer](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.5/Installer.exe)

**Comment by AL99117** [comment]
I downloaded the correct CUDA and cuDNN as required, and configured the environment variables, but I still used my CPU when using it. How to solve this? @fgilde 
![Uploading 微信图片_20241010115449.png…]()

**Reply by AL99117** [reply]
Initialized with provider Cpu
Failed to initialize provider DirectML. Message Unable to find an entry point named 'OrtSessionOptionsAppendExecutionProvider_DML' in DLL 'onnxruntime'.
Failed to initialize provider Cuda. Message [ErrorCode:RuntimeException] D:\a\_work\1\s\onnxruntime\core\session\provider_bridge_ort.cc:1637 onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error 126 "" when trying to load "C:\Users\Administrator\Desktop\AI\onnxruntime_providers_cuda.dll"

**Reply by fgilde** [reply]
Check if you have `C:\Program Files\NVIDIA\CUDNN\v9.4\bin\12.6` added to path. this was my main missing part

![image](https://github.com/user-attachments/assets/07ae1bb5-517a-4a60-97f3-c03584b632e1)

Additionally I Created a check cuda powershell script that can check if you have mostly installed everything correctly 
[check_cuda.zip](https://github.com/user-attachments/files/17324018/check_cuda.zip)

**Reply by AL99117** [reply]
To solve this problem, when installing cuDNN, the files should be copied to the corresponding installation directory of CUDA。Thank you！！

**Comment by AL99117** [comment]
Hello author, the current program only supports models trained based on Yolov 8n. Can you add support for the latest Yolov11 model or other models?

**Reply by fgilde** [reply]
Not sure but I can try. Do you have a sample model for 640x640 images that I can use to test?

**Comment by fgilde** [comment]
One more new version released

## Version Switch Support
 In the settings menu you can now switch between CUDA and DirectML versions

## Better human behavior
 - Simulate better delays for mouse click to prevent auto bans by time messure

## HWID Spoofer
 -  Added SecHex Spoofy under the tools menu to change HardwareID this can help if your PC is hardware banned
 
## 2 New drawing methods for ESP 
 - Added Media Drawing Context with VisualHost as new fastest and safe method (Not captureable)
 - Added GDI Context draw like the Desktop draw method on overlay (Also safe and not captureable)
 
 # Download

[CUDA VERSION](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.6/1.0.0.6_cuda.zip)
[Default DirectML Version](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.6/1.0.0.6.zip)
[Installer](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.6/Installer.exe)

**Reply by ZXCURSEDS** [reply]
Cant Download

**Reply by fgilde** [reply]
https://github.com/fgilde/AI-Ming/releases/tag/1.0.0.7

**Comment by H4x0r-exe** [comment]
First of all, English is not my native language, so I always use a translator. My question is whether Aimmy would work with the mouse's Y-axis inverted in games. I've tried to configure it several times, and each day it was a different setup, testing for long hours, but the result was always the same! When I press the right mouse button while aiming at an enemy, the Aimmy alignment quickly moves toward them and then pulls me down and to the sides, both left and right. I've tried reducing the effects to play with the aim assist, but it's not the same as using the mouse in its default position. So, back to my initial question: why doesn’t the aimbot work with an inverted axis? It may seem like a strange question, but I think there’s a way around this; after all, I can't use the normal axis like you do. Any ideas? Thank you in advance!

**Comment by venomissionary** [comment]
Is there any option to switch gpu 1. I think it running on my gpu 0 while gaming is affecting my 1% lows

**Comment by ZXCURSEDS** [comment]
possible to use the right stick of a gamepad as a movement method, instead of a mouse, for aim assist and aim config ?

**Reply by Nuzqek** [reply]
did you find out?

**Reply by q0pk** [reply]
use reWASD and mouse movement in aimmy

**Reply by bidikman** [reply]
comment on fait jarrive pas

**Reply by bidikman** [reply]
ya t il une version ou on peut utiliser la mantte ps4 ou ps5??? chez moi rien ne fonctionne

**Reply by hakanmadencioglu** [reply]
> PS4 veya PS5'i kullanan bir sürüm veya sürüm var mı??? chez moi rien ne fonctionne

lutfen ps5 uyumlu surum.. ps5 deki goruntuyu capture cart ile pc getirim.. pcde aimbot aktif edip ps5 ile oynamak harika olurdu

**Comment by hakanmadencioglu** [comment]
> Herkese selam,
> 
> Bu depoyu 2 gün önce buldum ve uzun zaman önce benzer bir proje oluşturduğum için çok ilginç buldum. Birkaç değişiklik ve iyileştirme yaptım, bunlar arasında şunlar yer alıyor:
> 
> Oyun Kumandası Desteği:
> 
> * Oyun kumandası okuma desteği eklendi.
> * Gamepad gönderme desteği üzerinde çalışılıyor.
> 
> Genel İyileştirmeler:
> 
> * Gösterge ve açma/kapama tuşu içeren global bir aktif anahtar eklendi.
> * Oyun kumandasının basılı olarak sayılması için ayarlar eklendi.
> * FOV Şeffaflık kaydırıcısı oluşturuldu.
> 
> Otomatik Tetikleme Geliştirmeleri:
> 
> * AIManager'da ayrı merkez ve kafa alanı algılama özelliğiyle otomatik tetiklemeyi tamamen Aim Assist'ten bağımsız hale getirdim.
> * Tetikleyiciye ek bir Gönder komutu eklendi, bu sayede düşmanların otomatik olarak işaretlenmesi sağlanıyor.
> * Tetikleme anını ayarlamak için özel kafa bölgesi ayarı yapabilirsiniz. Bu özellik ile bacak atışlarını da yapılandırabilirsiniz.
> * Tetikleyicinin çalıştırılmasından önce tetikleyici tuşuna basılı tutulması gereken minimum süreyi belirtme seçeneği.
> 
> ESP Geliştirmeleri:
> 
> * Tetik başlığının göreceli alanını gösterme seçeneği eklendi.
> 
> Bu proje üzerinde sadece 3 gündür çalışıyorum, bu yüzden şu an hiçbir şey mükemmel değil. Ama gelecekte şunları yapmayı planlıyorum:
> 
> Tetikleme sistemini tamamen değiştirerek birden fazla tetiklemeye izin verin. Bu, otomatik tetikleyici eklemenize, gönderilecek komutu ayarlamanıza, alanı tanımlamanıza, tuş atamalarını belirlemenize vb. olanak tanıyarak kapsamlı bir iş akışı oluşturmanızı sağlar. Otomatik hızlı ateşleme ekleyin.
> 
> Ancak, mevcut sürümüm şöyle görünüyor:
> 
> ![resim](https://private-user-images.githubusercontent.com/11070717/346184408-e9fbabce-4be2-49e2-a100-163d62b4e26a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5OTgwNzAsIm5iZiI6MTc3NTk5Nzc3MCwicGF0aCI6Ii8xMTA3MDcxNy8zNDYxODQ0MDgtZTlmYmFiY2UtNGJlMi00OWUyLWExMDAtMTYzZDYyYjRlMjZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDEyVDEyNDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3N2E3MjdlM2ZlNTYwYmVjMThmZWJmNzgzNjdkMGViZGVkOTk4NTIzZTU5YTMyYTRlNmZhODJhOGJlYTA0ODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.7mN8M46B0-iSWehgs0b3vh7CXJ_q5jzRNXHiZAJlaFc)
> 
> ![resim](https://private-user-images.githubusercontent.com/11070717/346184065-4ff4f0b1-ce6a-493c-8d76-a1bee6c5eac8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5OTgwNzAsIm5iZiI6MTc3NTk5Nzc3MCwicGF0aCI6Ii8xMTA3MDcxNy8zNDYxODQwNjUtNGZmNGYwYjEtY2U2YS00OTNjLThkNzYtYTFiZWU2YzVlYWM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDEyVDEyNDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5NTBlYTZjMWRhZTRjZGQyZDdiYWMxZDI5YTY1NTJkMTEzNmNiMzQwYjUxOTQ0ZDk1NWQ2NDAzMGFlMjIwMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9._hgiUBTznuwfS-3qROzL3uhKqF4t2sVKd8F1Iy8pYL0)
> 
> ![resim](https://private-user-images.githubusercontent.com/11070717/346184242-2e0d6838-9f3c-4266-918e-25c6f45b1f7e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5OTgwNzAsIm5iZiI6MTc3NTk5Nzc3MCwicGF0aCI6Ii8xMTA3MDcxNy8zNDYxODQyNDItMmUwZDY4MzgtOWYzYy00MjY2LTkxOGUtMjVjNmY0NWIxZjdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDEyVDEyNDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxZTJiNWM1MzUxZTFjZDU1NGJiMDUzZmVhYzQzMWRmOGJkMDkzZDU1NDdjNzJhNWM5Y2M4MzE4ZGU0YTljZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.ZotehS_Eyr5Vk_DiMxdUc6_N0Te7Wdr7DrTNKSzNwpM)
> 
> ![resim](https://private-user-images.githubusercontent.com/11070717/346006051-d466023e-a489-40a4-975f-54eec335314b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5OTgwNzAsIm5iZiI6MTc3NTk5Nzc3MCwicGF0aCI6Ii8xMTA3MDcxNy8zNDYwMDYwNTEtZDQ2NjAyM2UtYTQ4OS00MGE0LTk3NWYtNTRlZWMzMzUzMTRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDEyVDEyNDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNThhMmM0MmIzNDQ0OTYyZDI5NjRjMTE1M2I0M2EzNzM1NGVmMGU5MGMyZDE5YTE4Zjc1OGZiNmZiYTMzYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.dzvc2BUlNd_5mXBWol2GUhjFeuE6Ea_C05rV9hAChCE)
> 
> Ah... ve aktif olduğunda şöyle görünüyor, bu yüzden şu anda çalışıp çalışmadığını görmek kolay.
> 
> ![resim](https://private-user-images.githubusercontent.com/11070717/346005950-3b2a952f-dbce-4ce2-919b-dbdca9808951.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5OTgwNzAsIm5iZiI6MTc3NTk5Nzc3MCwicGF0aCI6Ii8xMTA3MDcxNy8zNDYwMDU5NTAtM2IyYTk1MmYtZGJjZS00Y2UyLTkxOWItZGJkY2E5ODA4OTUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDEyVDEyNDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2ODNkNDhjOTU5NThhNTcxNWZiOTRmMjJlYWUwOWZhMjJmMWZmMDQ5OTkyNDJhNWEwNTBlODlkNjE2YTk3NTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.eCDcD8NkvaDE-M0y7ALHw9rNuAlAt4FyRedN2h8r7qs)
> 
> Test etmek isterseniz, mevcut durumu burada bulabilirsiniz. Kullanmak isterseniz, DirectML.dll ve onnxruntime.dll dosyalarını bu dosyayı açtığınız dizine kopyaladığınızdan emin olun.
> 
> [TotallyNotAimmyV2_Ext.zip](https://github.com/fgilde/AI-Ming/releases/download/1.0.0.1/1.0.0.1.zip)
> 
> Genel sorum ana ekibe yöneliktir ve@BabyhamstaBu değişiklikleri siz de ister misiniz? O zaman biraz düzenleme yapıp bir fork PR oluşturmayı deneyeceğim.
> 
> Bu arada ve konu dışı: GitHub'da sana yıldız verdim, belki ana projem için de aynısını yapmak istersin ;-) https://github.com/fgilde/MudBlazor.Extensions/

peki nasil controller destekli oynayacagiz.. hic bir indirme mevcut degil

**Reply by apexou66-gif** [reply]
The link is broken.

**Comment by fgilde** [comment]
New version with OCR Region support and AutoPlay is available under https://github.com/fgilde/AI-Ming

**Comment by venomissionary** [comment]
Is it possible to run this on a 2nd gpu, I noticed input delay when using
the main gpu that is also used to play said game.

**Reply by fgilde** [reply]
sure deviceId is available as parameter, should be easy. I will add this feature in the next 2.0.0.2 release

**Reply by fgilde** [reply]
is donee, you now have a gpu select in the top right corner (v 2.0.0.2)

**Comment by venomissionary** [comment]
Awesome. Thanks!

**Comment by pcmamie2k24-a11y** [comment]
its work with playstation remote play ? like chiaki ? ty
